kbase-extension/static/kbase/js/common/ui.js

```diff
@@ -787,19 +833,23 @@ define([
             if (arg.classes) {
                 buttonClasses = buttonClasses.concat(arg.classes);
             }
-            if (!arg.event) {
-                arg.event = {};
-            }
 
             const attribs = {
                 type: 'button',
                 class: buttonClasses.join(' '),
-                title: title,
+                title: arg.title || arg.tip || arg.label,
                 dataButton: arg.name,
-                id: addButtonClickEvent(events, arg.event.type || arg.name, arg.event.data),
                 style: arg.style,
             };
 
+            if (events && arg.event) {
```

Comment:
make adding events optional -- only happens if both `events` and `arg.event` are present